### ISSUE TITLE
element summary in decoDesc

### ISSUE THREAD

**Issue by DariaElagina** [BetaMasaheft/Documentation#1038]
According to the[ guidelines](http://betamasaheft.eu/Guidelines/?q=decoration&id=decorationDescription), one can encode general observations on the decoration in the summary element inside the decoDesc.
So, I did it like that:
 ```xml
<decoDesc>
                        <summary>
                          In the upper part and in the middle of the scroll there are various magic drawings (square faces).
                        </summary>
                      </decoDesc>
```
However, Atom prohibts me to encode like that, and quilifies this as an error: element `decoDesc ` incomplete; missing required element `decoNote`.
Have I done something wrong? Is there some inconsistency between the schema and the guidelines?

**Comment by PietroLiuzzo** [comment]
yes, you are right, the guidelines were deceiving, as I cut down the example leaving decoDesc around summary. In the edit above I have fixed this, and additionally I declared the provenance of the example and clarified the introductory sentence, so that instead of multiple elements it explicitly says multiple `decoNote` elements. Otherways the schema is right, you are not allowed to have a `summary` there without having at least one `decoNote`.
